TAF_EasyRest/TestFramework/Test/BaseTest.cs

```diff
@@ -11,6 +14,8 @@ public void UserLogin(IWebDriver driver, string userEmail, string password)
             signInPage.SendKeysToEmailField(userEmail)
                 .SendKeysToPasswordField(password)
                 .ClickSignInButton();
+            string currentUrl = driver.Url;
+            new WebDriverWait(driver, TimeSpan.FromSeconds(3)).Until(d => driver.Url != currentUrl);
```

Comment:
I think it is better to have such functionality in the extension method

Reply:
Working on that...

Reply:
Changed.